doc/api/n-api.md

```diff
@@ -145,6 +145,12 @@ available to the module code.
 
 ## N-API Version Matrix
 
+N-API versions are additive NOT SemVer. Version 4 is an extension
+to version 3 in that is has all of the APIs from version 3 with
+some additions. This means that you
+do NOT need to recompile for new versions of Node.js which are
```

Comment:
If you want emphasis, I'd prefer we use _italics_ or **bold** rather than ALL CAPS. I'd prefer nothing, but if we're going to use emphasis, let's at least stick to the typographical conventions in use and not add to them. (Admittedly, there may be some ALL CAPS in the docs in a few places, but I'd like to avoid that one.)

```suggestion
do not need to recompile for new versions of Node.js which are
```